templates/guides/get-started.md

```diff
@@ -0,0 +1,208 @@
+---
```

Comment:
noticed i probably commented on the wrong part, the generated part and not this part but the comments all apply to these files i guess